Ubuntu-clang-latest/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM cgal/testsuite-docker:ubuntu
+
+MAINTAINER Maxime Gimeno
+
+#install the necessary packages
+RUN apt-get update \
+ && apt-get install -y python \
+ && apt-get install -y subversion 
```

Comment:
Add `apt-get clean -y` at the end, to remove the apt caches, and reduce the size of the image.